www/index.php

```diff
@@ -94,9 +99,9 @@ function default_status_time() {
  */
 foreach ($config['feature'] as $feature) {
     if ($feature['enabled'] == "on") {
-        error_log("Including Feature {$feature['name']}");
-        include $feature['name'] . '/lib.php';
-        include $feature['name'] . '/routes.php';
+        $log->debug("Including Feature {$feature['name']}");
```

Comment:
We're a bit behind the docs online. I found this helpful 
https://github.com/etsy/morgue/blob/master/vendor/slim/slim/docs/logging.markdown